blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AbstractMetricsRW.java

```diff
@@ -52,8 +53,16 @@
             CacheBuilder.newBuilder().expireAfterAccess(10,
                         TimeUnit.MINUTES).concurrencyLevel(16).build();
 
+    // this collection is used to reduce the number of delayed locators that get written per slot. Simply, if a locator
+    // has been seen for a slot, don't bother.
+    private static final Cache<String, Boolean> insertedDelayedLocators = CacheBuilder.newBuilder().expireAfterAccess(10,
+            TimeUnit.MINUTES).concurrencyLevel(16).build();
+
     private static final Logger LOG = LoggerFactory.getLogger(AbstractMetricsRW.class);
 
+    protected boolean isTrackingDelayedMetrics;
```

Comment:
When I named it some kinda tracking seemed appropriate. I thought this would track it in the database whereas the one which is already existing would track it in the logs. Also the actual config is `ENABLE_TRACKING_DELAYED_METRICS`

The existing config is `TRACKER_DELAYED_METRICS_MILLIS`.